roles/openshift_facts/library/openshift_facts.py

```diff
@@ -393,6 +393,9 @@ def set_metrics_facts_if_unset(facts):
         if 'use_cluster_metrics' not in facts['common']:
             use_cluster_metrics = False
             facts['common']['use_cluster_metrics'] = use_cluster_metrics
+        if 'use_cluster_metrics_hawkular' not in facts['common']:
```

Comment:
Later would be fine for that one. it would at least cut down on user confusion as to what magic creates the services.